ReactSkia/pluginfactory/RnsPluginFactory.cpp

```diff
@@ -31,6 +31,24 @@ RnsPluginFactory::RnsPluginFactory(){
     RNS_LOG_ERROR(" Platform Application Manager Plugin Loading Error :  " << ex.what());
   }
 #endif // RNSP_APP_MANAGER
+
+#if ENABLE(RNSP_PLATFORM_MANAGER)
+  RNS_LOG_INFO("========== Loading Platform Manager Plugin : " << RNSP_PLATFORM_MANAGER_PLUGIN << " for RNS ==========");
```

Comment:
Is this kept as INFO explicitly for development purpose ?, if not change it to DEBUG.
Same for App Manger Plugin Aswell